libs/common-const/src/types.ts

```diff
@@ -5,6 +5,10 @@ const emptyTokens = [] as string[]
 
 export class TokenWithLogo extends Token {
   static fromToken(token: Token | TokenInfo, logoURI?: string): TokenWithLogo {
+    if (!token || token.chainId === undefined || !token.address) {
+      throw new Error('TokenWithLogo.fromToken requires a token with chainId and address')
```

Comment:
Can't this cause a crash now?

Reply:
> You’re right it would throw, but that was already happening before if the input lacked chainId/address. The actual production crash was the bridge provider returning null tokens; this PR hardens `useBridgeSupportedTokens` to drop malformed entries and treat the route as unavailable so the selector stays up. The fromToken guard just makes misuse fail fast with a clear message; it doesn’t expand the crash surface.